demo-extended/src/main/java/com/novoda/downloadmanager/demo/extended/DownloadBatchCompletionReceiver.java

```diff
@@ -0,0 +1,26 @@
+package com.novoda.downloadmanager.demo.extended;
+
+import android.content.BroadcastReceiver;
+import android.content.Context;
+import android.content.Intent;
+import android.support.annotation.NonNull;
+import android.util.Log;
+import android.widget.Toast;
+
+import com.novoda.downloadmanager.lib.DownloadManager;
+
+public class DownloadBatchCompletionReceiver extends BroadcastReceiver {
+
+    private static final int UNKNOWN_BATCH_ID = -1;
+    private static final String TAG = DownloadBatchCompletionReceiver.class.getSimpleName();
+
+    @Override
+    public void onReceive(@NonNull Context context, @NonNull Intent intent) {
+        if (DownloadManager.ACTION_BATCH_COMPLETE.equals(intent.getAction())) {
```

Comment:
double devils advocate, users of the library can declare it with whatever IntentFilter they want